src/amo/components/RatingsByStar/index.js

```diff
@@ -29,13 +28,37 @@ type Props = {|
 
 type InternalProps = {|
   ...Props,
+  clientApp: string,
   dispatch: DispatchFunc,
   errorHandler: ErrorHandlerType,
   groupedRatings?: GroupedRatingsType,
   i18n: I18nType,
   location: ReactRouterLocationType,
+  siteLang: string,
 |};
 
+type WrappingLinkProps = {|
+  styleClass: string,
+  children: React.Node,
+  rowProps: {
+    title: string,
+    link: string,
+  },
+|};
+
+function WrappingLink(props: WrappingLinkProps) {
+  const { styleClass, children, rowProps } = props;
+
+  if (Object.keys(rowProps).length > 0) {
```

Comment:
I gather what you're dealing with here is the case when we're either `loading` or there is no `addon`? Checking `rowProps` feels a bit obscure for that, so maybe there's a better way of checking for these than that?